### PR TITLE
register

### PR DESCRIPTION
**Pull request by hieupc-0248**
## Related Tickets:
- [#68058](https://edu-redmine.sun-asterisk.vn/issues/68058)

## WHAT this PR does?
- sửa giao diện đăng kí, validate các trường

## Notes Impacted Areas:
- app/Http/Controllers/Auth/RegisteredUserController.php
- resources/views/auth/register.blade.php
- app\Http\Requests\Auth\RegisterRequest.php
## Attachments:
- 
image

### CODE REVIEW

**Comment by anhcnn-0378** [comment]
Oke

**Comment by dungpt-0371** [comment]
oke

**Comment by hieupc-0248** [comment]
ready

File: app/Http/Controllers/Auth/RegisteredUserController.php [hunk 1]
```diff
@@ -31,30 +30,23 @@ public function create()
...
         $user->phone = $request->phone;
         $user->address = $request->address;
         $user->is_active = true;
-        $user->role_id = 1;
+        $user->role_id = 2;
         $user->email = $request->email;
         $user->password = Hash::make($request->password);
         $user->save();
-
+
```
Comment (ghost) [inline app/Http/Controllers/Auth/RegisteredUserController.php]: delete spaces

File: app/Http/Controllers/Auth/RegisteredUserController.php [hunk 2]
```diff
@@ -31,30 +30,23 @@ public function create()
...
-        ]);
-
         $user = new User();
         $user->first_name = $request->first_name;
         $user->last_name = $request->last_name;
         $user->phone = $request->phone;
         $user->address = $request->address;
         $user->is_active = true;
-        $user->role_id = 1;
+        $user->role_id = 2;
```
Comment (ghost) [inline app/Http/Controllers/Auth/RegisteredUserController.php]:
```suggestion
        $user->role_id = Role::where(...)->first();
```

File: resources/views/auth/register.blade.php [hunk 3]
```diff
@@ -56,4 +72,4 @@
             </div>
         </form>
     </x-auth-card>
-</x-guest-layout>
+</x-guest-layout>
```
Comment (ghost) [inline resources/views/auth/register.blade.php]: eof

**Comment by hieupc-0248** [comment]
ready

**Pull request merged**